File: Programmers/[12914] 멀리 뛰기/Johoseong/12914.py
def solution(n):
    if n == 1:
        return 1
    jumps = [0 for _ in range(n + 1)]
    jumps[1] = 1
    jumps[2] = 2

    for i in range(3, n + 1):
        jumps[i] = jumps[i - 1] + jumps[i - 2]

    return jumps[n] % 1234567

File: Programmers/[12914] 멀리 뛰기/woong-jae/12914.js
function solution(n) {
    const dp = Array(n + 1).fill(0);
    dp[0] = 1;
    for(let i = 0; i < n; i++) {
        if(i + 1 <= n) dp[i + 1] = (dp[i + 1] + dp[i]) % 1234567;
        if(i + 2 <= n) dp[i + 2] = (dp[i + 2] + dp[i]) % 1234567;
    }
    return dp[n];
}